molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/data/meta/MetaTestConfig.java

```diff
@@ -0,0 +1,50 @@
+package org.molgenis.integrationtest.data.meta;
+
+import org.molgenis.data.DataService;
+import org.molgenis.data.EntityFactoryRegistrar;
+import org.molgenis.data.EntityFactoryRegistry;
+import org.molgenis.data.EntityManagerImpl;
+import org.molgenis.data.i18n.SystemEntityTypeI18nInitializer;
+import org.molgenis.data.listeners.EntityListenersService;
+import org.molgenis.data.meta.DefaultPackage;
+import org.molgenis.data.meta.EntityTypeDependencyResolver;
+import org.molgenis.data.meta.MetaDataService;
+import org.molgenis.data.meta.MetaDataServiceImpl;
+import org.molgenis.data.meta.model.AttributeFactory;
+import org.molgenis.data.meta.persist.PackagePersister;
+import org.molgenis.data.meta.system.*;
+import org.molgenis.data.meta.util.AttributeCopierImpl;
+import org.molgenis.data.meta.util.EntityTypeCopierImpl;
+import org.molgenis.data.platform.decorators.SystemRepositoryDecoratorRegistryImpl;
+import org.molgenis.data.security.SystemEntityTypeRegistryImpl;
+import org.molgenis.data.support.DataServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.annotation.PostConstruct;
+
+@Configuration
+@Import({ DataServiceImpl.class, MetaDataServiceImpl.class, EntityManagerImpl.class,
+		SystemRepositoryDecoratorRegistryImpl.class, EntityFactoryRegistry.class, EntityListenersService.class,
+		AttributeFactory.class, SystemEntityTypeRegistryImpl.class, EntityTypeDependencyResolver.class,
+		SystemPackageRegistry.class, PackagePersister.class, SystemEntityTypePersister.class,
+		SystemEntityTypeRegistrar.class, SystemPackageRegistrar.class, EntityFactoryRegistrar.class,
+		SystemEntityTypeInitializer.class, SystemEntityTypeI18nInitializer.class, DefaultPackage.class,
+		EntityTypeCopierImpl.class, AttributeCopierImpl.class })
+@ComponentScan("org.molgenis.data.meta.model")
```

Comment:
Is this component scan required? If yes, can you add a note why. What is the relationship with org.molgenis.data.config.MetadataTestConfig?

Reply:
Can you create a technical debt issue to deal with the unclarified issues later?

Reply:
Issue #6848